app/addons/fauxton/assets/scss/_navigation.scss

```diff
@@ -136,6 +134,16 @@
   border-radius: 50%;
 }
 
+.faux-navbar--wide .faux-navbar__icon-badge:after {
+  bottom: 12px;
+  right: 107px;
+}
+
+.faux-navbar--narrow .faux-navbar__icon-badge:after {
+  bottom: 38px;
+  right: 10px;
+}
+
```

Comment:
`faux-navbar--narrow` and `faux-navbar--wide` are defined around line 46, better to add these extra rules there to keep them together.